paws.common/R/iniutil.R

```diff
@@ -0,0 +1,33 @@
+# Get the profile name from an ini file
+extract_ini_profile <- function(item) {
+  profile <- gsub("\\[|\\]", "", item)
+  return(profile)
+}
+
+# Get a parameter and its value
+extract_ini_parameter <- function(item) {
+    item_split <- strsplit(item, "=")[[1]]
```

Comment:
I think this should split at the first `=`, since this might split into more parts if someone has a `=` in the value.